.travis.yml

```diff
@@ -1,11 +1,58 @@
-os:
-  - linux
 language: cpp
-compiler:
-  - gcc
-  - clang
+
+cache:
+  directories:
+  - ${ExternalData_OBJECT_STORES}
+
+env:
+  global:
+    - ExternalData_OBJECT_STORES="${HOME}/.ExternalData"
+    - PROJ_SRC="${TRAVIS_BUILD_DIR}"
+    - ITK_MODULE_NAME=MetaIO
+    - ITK_BUILD_MODULES="ITKMetaIO;ITKIOMeta"
+    - ITK_TAG=master
```

Comment:
True, but the matrix would start becoming quite big as I did not find a solution to "summarize" the matrix options, but had to list all individual settings.